projects/dotcms-field-elements/src/components/dot-time/dot-time.e2e.ts

```diff
@@ -1,131 +1,399 @@
 import { E2EElement, E2EPage, newE2EPage } from '@stencil/core/testing';
 import { EventSpy } from '@stencil/core/dist/declarations';
+import { dotTestUtil } from '../../utils';
 
 describe('dot-time', () => {
     let page: E2EPage;
     let element: E2EElement;
-    let input: E2EElement;
-    let spyStatusChangeEvent: EventSpy;
-    let spyValueChange: EventSpy;
-
-    beforeEach(async () => {
-        page = await newE2EPage({
-            html: `
-              <dot-time
-                    label="Time:"
-                    name="time01"
-                    value="18:30:30"
-                    hint="Time hint"
-                    required
-                    required-message="Required Time"
-                    validation-message="Time out of range"
-                    min="06:00:00"
-                    max="22:00:00"
-                    step="10"
-                ></dot-time>`
-        });
-
-        spyStatusChangeEvent = await page.spyOnEvent('statusChange');
-        spyValueChange = await page.spyOnEvent('valueChange');
-        element = await page.find('dot-time');
-        input = await page.find('input');
-    });
+    let inputCalendar: E2EElement;
 
-    it('should render', () => {
-        // tslint:disable-next-line:max-line-length
-        const tagsRenderExpected = `<dot-time label=\"Time:\" name=\"time01\" value=\"18:30:30\" hint=\"Time hint\" required=\"\" required-message=\"Required Time\" validation-message=\"Time out of range\" min=\"06:00:00\" max=\"22:00:00\" step=\"10\" class=\"dot-valid dot-pristine dot-untouched dot-required hydrated\"><div class=\"dot-field__label\"><label for=\"dot-time01\">Time:</label><span class=\"dot-field__required-mark\">*</span></div><dot-input-calendar type=\"time\" required-message=\"Required Time\" validation-message=\"Time out of range\" class=\"hydrated\"><input id=\"dot-time01\" required=\"\" type=\"time\" min=\"06:00:00\" max=\"22:00:00\" step=\"10\"></dot-input-calendar><span class=\"dot-field__hint\">Time hint</span></dot-time>`;
-        expect(element.outerHTML).toBe(tagsRenderExpected);
-    });
+    describe('render CSS classes', () => {
+        beforeEach(async () => {
```

Comment:
Check the `beforeEach` because I think they're repeating in each describe and we could extract into one.